packages/ui-dom-utils/src/getComputedStyle.ts

```diff
@@ -35,16 +36,20 @@ import { canUseDOM } from './canUseDOM'
  * specified element
  * @module getComputedStyle
  *
- * @param {ReactComponent|DomNode} el - component or DOM node
- * @returns {Object} object containing css properties and values for the element
+ * @param { Node | Window | React.ReactElement | function } el - component or DOM node
+ * @returns { Object } object containing css properties and values for the element
  */
-// @ts-expect-error ts-migrate(7006) FIXME: Parameter 'el' implicitly has an 'any' type.
-function getComputedStyle(el) {
-  let style = {}
+function getComputedStyle(
+  el: Node | Window | React.ReactElement | ((...args: any[]) => any)
+) {
+  let style: CSSStyleDeclaration | Record<string, never> = {}
```

Comment:
I think we can safely type this to CSSStyleDeclaration, it wont change its behaviour. So instead of defaulting to `{}` new can write new CSSStyleDeclaration()

Reply:
`new CSSStyleDeclaration` is `Illegal constructor`, so we have to keep the empty object for now